### PR TITLE
summary.js: use double quotes in the `a` tag.

### PR DESCRIPTION
**Pull request by XhmikosR**
No functional change.

### CODE REVIEW

**Review by osk2 — APPROVED** [review]

**Comment by MaledongGit** [comment]
Any special reasons？

File: scripts/helpers/summary.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ module.exports = (contents, locale, path) => {
     .not((i, elem) => IGNORE_SELECTORS.some((selector) => $(elem).is(selector)))
     .each((i, elem) => {
       if (summary.length > SUMMARY_MAX_LENGTH) {
-        summary += `<p><a href='/${locale}/${path.replace(/\\/g, '/')}/'>Read more...</a></p>`
```
Comment (MaledongGit) [inline scripts/helpers/summary.js]: Any special reasons？

File: scripts/helpers/summary.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ module.exports = (contents, locale, path) => {
...
+        summary += `<p><a href="/${locale}/${path.replace(/\\/g, '/')}/">Read more...</a></p>`
```
Comment (ghost) [inline scripts/helpers/summary.js:33]: Any special reasons？

Comment (XhmikosR) [reply scripts/helpers/summary.js:33]: Consistency. There's no other HTML attribute which uses single quotes AFAICT.

**Review by Trott — APPROVED** [review]

**Pull request merged**